pkg/raft/node.go

```diff
@@ -124,468 +114,15 @@ func IsEmptySnap(sp pb.Snapshot) bool {
 	return sp.Metadata.Index == 0
 }
 
-// Node represents a node in a raft cluster.
-type Node interface {
-	// Tick increments the internal logical clock for the Node by a single tick. Election
-	// timeouts and heartbeat timeouts are in units of ticks.
-	Tick()
```

Comment:
Might want to double-check that the `RawNode` comments for these methods are not missing details that might be present in this interface's comments, and "migrate" the missing bits over if any.

Reply:
There are indeed a bunch of comments that can be copied over to `RawNode`. I'll send another PR with updated `RawNode` comments, will need to revisit each one.

Reply:
#132006